packages/edge-demo-site/README.md

```diff
@@ -1,54 +1,7 @@
-# Welcome to Remix!
+# Remix Demo Site for Netlify Edge Functions
 
-- [Remix Docs](https://remix.run/docs)
-- [Netlify Edge Functions Overview](https://docs.netlify.com/netlify-labs/experimental-features/edge-functions)
+This is a demo site for [Remix](https://remix.run) apps running on
```

Comment:
Mind having a peek at this @stephmarie17? It's to the point, but just want to make sure about the wording. All the stuff I removed will be reworked into the actual Remix template.